fruit_classifier/predict/predict_utils.py

```diff
@@ -6,6 +6,54 @@
 from skimage.transform import resize
 
 
+def draw_text_with_background(output_image,
+                              text="text",
+                              text_offset_x=10,
+                              text_offset_y=25,
+                              font_scale=.7,
+                              font_thickness=2,
+                              font_color=(0, 0, 0),
+                              font_bg_color=(255, 255, 255),
+                              font=cv2.FONT_HERSHEY_SIMPLEX,
+                              padding=5):
+    """
+    Draws text inside a colored box ontop of an image
+
+    Parameters
+    ----------
+    output_image    : cv2 image -  The image
+    text            : str - The text
+    text_offset_x   : integer - text start position x axis
+    text_offset_y   : integer - text start position y axis
+    font_scale      : integer - font scale used
+    font_thickness  : integer - font thickness used
+    font_color      : integer array - text color
+    font_bg_color   : integer array - background box color
+    font            : cv2 font - font of text
+    padding         : integer - margin of the box around text
+    """
+
+    # get the width and height of the text box from the text
+    (text_width, text_height) = \
+        cv2.getTextSize(text, font, fontScale=font_scale,
+                        thickness=font_thickness)[0]
+
+    # make the coords of the box with a small padding
```

Comment:
Is the box needed?

Reply:
On some images, the text color is the same as the image in the background, and so the text is either hardly or not at all readable. With a black box behind the white text, the text is always readable.